HistoryEntityManager/src/main/resources/application.properties

```diff
@@ -11,11 +10,14 @@ mysettings.postgres.username=${dbuser:ngb}
 mysettings.postgres.password=${dbpass:ngb}
 mysettings.postgres.database-name=${dbname:ngb}
 mysettings.gateway.host=${gateway.host:localhost}
-mysettings.gateway.port=${gateway.port:9090}
-atcontext.url=http://${mysettings.gateway.host}:${mysettings.gateway.port}/ngsi-ld/contextes/
+mysettings.gateway.port=${gateway.port:1040}
+quarkus.http.port=${mysettings.gateway.port}
+atcontext.host=${atctxhost:localhost}
```

Comment:
please change this default value to at-context-server as this is the default in the distributed case compose files and this application properties will only be used by that. same on the other microservices .